everest

```diff
@@ -197,12 +197,12 @@ write_ocaml_env_dest_file () {
   str="
     # These lines automatically added by $0
     export PATH=/cygdrive/c/ocamlmgw64/bin/:\$PATH
-    export PATH=\$HOME/.opam/system/bin:\$PATH
-    export PATH=\$HOME/.opam/system/lib/stublibs:\$PATH
+    export PATH=\$HOME/.opam/$OPAM_VERSION/bin:\$PATH
```

Comment:
this should now be performed automatically by `opam init` (which the everest script will call) ... the whole function can go

Reply:
Fixed. That's awesome, did not know that.